pkg/distributor/tee_test.go

```diff
@@ -0,0 +1,47 @@
+package distributor
+
+import (
+	"testing"
+
+	"github.com/grafana/loki/pkg/push"
+	"github.com/stretchr/testify/mock"
+)
+
+type mockedTee struct {
```

Comment:
Nit, but there is already a `mockTee` here: https://github.com/grafana/loki/blob/111c19b38147837041c6cc7c7483abe00c0c1599/pkg/distributor/distributor_test.go#L1268

Reply:
yeah it's a bit different.